lib/components/Navigation/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { FlexBox } from "@/components/Layout/_FlexBox"
+import { Icon, IconType } from "@/components/Utilities/Icon"
+import { ChevronRight } from "@/icons"
+import { forwardRef } from "react"
+
+interface BreadcrumbsType {
+  icon: IconType
+  route: string
+  title: string
+}
+
+export const Breadcrumb = forwardRef<HTMLDivElement, BreadcrumbsType>(
+  ({ icon, route, title }, ref) => {
+    return (
+      <FlexBox
```

Comment:
Same thing about `FlexBox` I said before 👀 . Actually, if you look at the filename, it starts with `_` which tries to convey "private" 😆